### DISCUSSION TITLE
Can't install

### DISCUSSION THREAD

**Discussion by TheBlizZar** [LSPosed/MagiskOnWSALocal#646] [Q&A]
I tried to install the software but after i choose the differents options but i get this : 

Mount images
fuse.erofs 1.6-g689372cd
disk: /tmp/wsa-build-sB9QmhE37Q_/wsa/x64/system.img
offset: 0
mountpoint: /tmp/wsa-build-sB9QmhE37Q_/erofs
dbglevel: 0
<E> erofs: cannot find valid erofs superblock
failed to read erofs super block
Build: an error has occurred, exit

Can someone help me please ?

**Comment by Almighty188** [comment]
*** Note *** Build is broken per [Bug] version 2306 build failed https://github.com/LSPosed/MagiskOnWSALocal/issues/615 but can be built by manually making changes to scripts/build.sh with many thanks to @YT-Advanced as seen here:
https://github.com/LSPosed/MagiskOnWSALocal/pull/620/files
